bears/configfiles/TOMLBear.py

```diff
@@ -0,0 +1,30 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='tomlv',
+        use_stderr=True,
+        output_format='regex',
+        output_regex=r"Error in '.*': "
+                     r'Near line (?P<line>\d+) '
+                     r"\(last key parsed '.*'\): (?P<message>.*)")
+class TOMLBear:
+    """
+    Checks the code for formatting in ``TOML`` documents
+    and prints each key's type.
+
+    This is done using ``tomlv`` utility. For more information visit
+    <https://github.com/BurntSushi/toml/tree/master/cmd/tomlv>.
```

Comment:
Also, please address all the comments [here](https://github.com/coala/coala-bears/pull/1409#pullrequestreview-21414045)